langs/cpp/cpp.c

```diff
@@ -29,13 +30,22 @@ int main(int argc, char* argv[]) {
 
     fclose(fp);
 
+    // Before main runs, set stdout and stderr to unbuffered
+    fp = fopen(buffer_fix, "w");
+    fputs(
```

Comment:
Should I be checking error on these? I don't see how they could fail after code.cpp already worked

Reply:
It's not strictly necessary but it doesn't hurt. Exiting with an error is better than undefined behavior.